temp_workspace/input/path_config.yaml

```diff
@@ -0,0 +1,5 @@
+  pdb_dir: '../temp_workspace/input/'
```

Comment:
I thought we were going to get rid of a working directory... if it's something that only exists for the length of the simulation, perhaps it could be created and deleted with the `os` module

Reply:
This particular working directory exists only for demonstration purposes - this path would be changed out by the user. temp_workspace/ should not be in the PR - I'll look into addressing this after work today.

Reply:
Maybe:
- having the pdb data in a `data/` or `pdb_structures/` directory inside `simSPI/simSPI`
- having the config files at the root of the `simSPI/simSPI` repo
would be more intuitive?

We do not need to tell the users how to organize their workspace.

Reply:
And the user can call the simulator by providing a path `output_dir` to tell it where to store the output of the simulation?

Reply:
Good ideas, thank you!

I've edited the folder structure - temp_workspace is gone (as I mentioned earlier it's best not left in the main branch anyhow) and the config files now sit in the "root" simSPI/ directory. The PDBs are stored in simSPI/data as you suggested. 

For now, I think simSPI/output/ makes sense as a default output directory and the user can change it in the config file if desired.